ext/openssl/openssl_backend_common.c

```diff
@@ -1700,11 +1693,9 @@ void php_openssl_load_cipher_mode(struct php_openssl_cipher_mode *mode, const EV
 	int cipher_mode = EVP_CIPHER_mode(cipher_type);
 	memset(mode, 0, sizeof(struct php_openssl_cipher_mode));
 	switch (cipher_mode) {
-#if PHP_OPENSSL_API_VERSION >= 0x10100
-		/* Since OpenSSL 1.1, all AEAD ciphers use a common framework. We check for
-		 * EVP_CIPH_OCB_MODE, because LibreSSL does not support it. */
 		case EVP_CIPH_GCM_MODE:
 		case EVP_CIPH_CCM_MODE:
+		/* We check for EVP_CIPH_OCB_MODE, because LibreSSL does not support it. */
 # ifdef EVP_CIPH_OCB_MODE
```

Comment:
Minor nit:
```suggestion
#ifdef EVP_CIPH_OCB_MODE
```
As it was indented due to the prior `#if`, simiarly for `# ifdef NID_chacha20_poly1305`